src/dlgProfilePreferences.cpp

```diff
@@ -681,35 +681,34 @@ void dlgProfilePreferences::disconnectHostRelatedControls()
 {
     disconnect(buttonDownloadMap, SIGNAL(clicked()));
 
+    disconnect(pushButton_foreground_color, SIGNAL(clicked()));
+    disconnect(pushButton_background_color, SIGNAL(clicked()));
     disconnect(pushButton_command_line_foreground_color, SIGNAL(clicked()));
     disconnect(pushButton_command_line_background_color, SIGNAL(clicked()));
+    disconnect(pushButton_command_foreground_color, SIGNAL(clicked()));
+    disconnect(pushButton_command_background_color, SIGNAL(clicked()));
 
     disconnect(pushButton_black, SIGNAL(clicked()));
-    disconnect(pushButton_Lblack, SIGNAL(clicked()));
-    disconnect(pushButton_green, SIGNAL(clicked()));
-    disconnect(pushButton_Lgreen, SIGNAL(clicked()));
+    disconnect(pushButton_lBlack, SIGNAL(clicked()));
     disconnect(pushButton_red, SIGNAL(clicked()));
-    disconnect(pushButton_Lred, SIGNAL(clicked()));
-    disconnect(pushButton_blue, SIGNAL(clicked()));
-    disconnect(pushButton_Lblue, SIGNAL(clicked()));
+    disconnect(pushButton_lRed, SIGNAL(clicked()));
+    disconnect(pushButton_green, SIGNAL(clicked()));
+    disconnect(pushButton_lGreen, SIGNAL(clicked()));
     disconnect(pushButton_yellow, SIGNAL(clicked()));
-    disconnect(pushButton_Lyellow, SIGNAL(clicked()));
-    disconnect(pushButton_cyan, SIGNAL(clicked()));
-    disconnect(pushButton_Lcyan, SIGNAL(clicked()));
+    disconnect(pushButton_lYellow, SIGNAL(clicked()));
+    disconnect(pushButton_blue, SIGNAL(clicked()));
+    disconnect(pushButton_lBlue, SIGNAL(clicked()));
     disconnect(pushButton_magenta, SIGNAL(clicked()));
-    disconnect(pushButton_Lmagenta, SIGNAL(clicked()));
+    disconnect(pushButton_lMagenta, SIGNAL(clicked()));
+    disconnect(pushButton_cyan, SIGNAL(clicked()));
+    disconnect(pushButton_lCyan, SIGNAL(clicked()));
     disconnect(pushButton_white, SIGNAL(clicked()));
-    disconnect(pushButton_Lwhite, SIGNAL(clicked()));
-
-    disconnect(pushButton_foreground_color, SIGNAL(clicked()));
-    disconnect(pushButton_background_color, SIGNAL(clicked()));
-    disconnect(pushButton_command_foreground_color, SIGNAL(clicked()));
-    disconnect(pushButton_command_background_color, SIGNAL(clicked()));
+    disconnect(pushButton_lWhite, SIGNAL(clicked()));
 
     // The "new" style connect(...) does not have the same range of overloaded
     // disconnect(...) counterparts - so we need to provide the "dummy"
     // arguments to get the wanted wild-card behaviour for them:
-    disconnect(reset_colors_button, &QAbstractButton::clicked, 0, 0);
+    disconnect(pushButton_resetColors, &QAbstractButton::clicked, 0, 0);
```

Comment:
For some reason I hadn't spotted that those last two arguments should be `nullptr` like the following line - in pending commit